tests/test_orient_symmetric.py

```diff
@@ -401,3 +412,14 @@ def buildCyclicRotations(self, order):
         rots_symm = Rotation.from_euler(angles).matrices
 
         return rots_symm
+
+    def angle_dist(self, r1, r2):
```

Comment:
What is this calculating exactly?

Reply:
This uses the axis-angle representation of the rotation between `r1` and `r2`, ie `r1 @ r2.T`, to find the angular distance between `r1` and `r2`. I moved this to `.../utils/rotation.py` and added a docstring. Will add a test momentarily.

Reply:
Sounds good.

Reply:
Added a test for this method.